### ISSUE TITLE
Support Redmine 4.0.x

### ISSUE THREAD

**Issue by HungTDO** [HugoHasenbein/redmine_all_thumbnails#1]
PLz upgrade plugin to support Redmine 4.0.x. Thanks

**Comment by double2ugly** [comment]
yeah, i agree... that would be great... also for your other plugins...

**Comment by yennor** [comment]
I've got the same problem here. The plugins are not compatible with redmine 4
